packages/react-composites/src/composites/CallComposite/pages/CallPage.tsx

```diff
@@ -119,6 +128,8 @@ export const CallPage = (props: CallPageProps): JSX.Element => {
       dataUiId={'call-page'}
       latestErrors={props.latestErrors}
       onDismissError={props.onDismissError}
+      /* @conditional-compile-remove(gallery-layouts) */
+      onSetUserSetOverflowGalleryPosition={setUserSetOverflowGalleryPosition}
```

Comment:
`onSetUserSet`? 
Not convinced on userSet but if its staying, update this to `onUserSetOverflowGalleryPositionChange` to avoid the double set

Reply:
(naming was never my strong suit!)